src/main/resources/anniv5Resources/localization/eng/serpentinepack/Keywordstrings.json

```diff
@@ -7,7 +7,7 @@
   {
     "PROPER_NAME": "Cunning",
     "NAMES": ["cunning"],
-    "DESCRIPTION": "When you enter this Stance, gain #b2 Vigor for each #yunique debuff among ALL enemies."
+    "DESCRIPTION": "When you enter this Stance, gain #b3 Vigor for each #yunique debuff among ALL enemies."
```

Comment:
Didn't change the stance string to reflect this change... also didn't change the stance to do this.